pages/documentation/developer/tutorials/single-line-diagram/svg-writing.md

```diff
@@ -65,7 +65,7 @@ Here are some details about these dependencies (see also the [powsybl artifacts
 - `powsybl-single-line-diagram-core` is the core module of single-line-diagram,
```

Comment:
and then we should write that powsybl-starter includes some basic powsybl dependencies, the one really needed being: (...)

Reply:
powsybl-config-test is included in the svg writing pom but not in the powsybl-starter pom. Shall we mention that it should be added in the starter pom ?

Reply:
Discussed together: powsybl-config-test is not necessary anymore since PR https://github.com/powsybl/powsybl-core/pull/2062